### DISCUSSION TITLE
Directus 9.21.0 development broken after upgrade

### DISCUSSION THREAD

**Discussion by kevincam3** [directus/directus#16529] [Archive]
Here's what I get after this last update.

![image](https://user-images.githubusercontent.com/870934/202847353-92969e99-371f-4b40-aa35-667e317816c7.png)

**Comment by paescuj** [comment]
Cannot reproduce on my end. Did you remember to run `pnpm install` & `pnpm -r build` beforehand? See https://docs.directus.io/contributing/running-locally.html#_4-install-the-dependencies-and-build-the-project

**Reply by kevincam3** [reply]
I didn't because I've been running "dev" but I did run it and got a different error.
ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @directus/app@9.21.0 build: `vite build`

![image](https://user-images.githubusercontent.com/870934/202849072-30c65fbe-65e6-43f1-b660-1614df3e0e63.png)

**Reply by kevincam3** [reply]
Ok, I think I'm good. I see why that error happened and it's not Directus related. I just had too many things running and ran out of memory so after running "dev" after doing the "build" things are working normal again. Thanks for the help. I'm adding that to my upgrade notes for the next time.

**Reply by paescuj** [reply]
> I didn't because I've been running "dev" [...]

Ah yeah you need to run those commands after fetching a new state from GitHub 👍 

> Ok, I think I'm good. I see why that error happened and it's not Directus related. I just had too many things running and ran out of memory so after running "dev" after doing the "build" things are working normal again. Thanks for the help. I'm adding that to my upgrade notes for the next time.

Nice! I'm glad it's working now! 😃
